packages/instant/src/components/ui/overlay_container.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { generateMediaWrapper, ScreenWidths } from '../../style/media';
+import { styled } from '../../style/theme';
+import { zIndex } from '../../style/z_index';
+
+import { Container, ContainerProps } from './container';
+
+interface OverlayProps {
+    showMaxWidth?: ScreenWidths;
+}
+const Overlay =
```

Comment:
When I first started out I was using an `:after` selector, but this has evolved into looking similar enough that it might make sense to modify the existing `Overlay` component instead of generating this new one.

Reply:
This might make sense as a refactor once https://github.com/0xProject/0x-monorepo/pull/1230 is merged, as 1230 is what makes the more generic Overlay